frontend/README.md

```diff
@@ -0,0 +1,117 @@
+# Frontend
+
+
+## Structure
+
+```bash
+# One folder per (micro)frontend.
+{service-name}/
+    # Compiled result of yarn build is stored in this folder
+    dist/
+    # Modules
+    node_modules/
+    # These files will be public, but other public files are added by webpack
+    # Normally, you will not have to add files here
+    public/
+        # Index file, modified by webpack during compilation to include the compiled sources in /src
+        index.html
+    src/
+        # Images, fonts, and other assets only used by this service.
+        assets/
+        # Contains all the service specific vue components, optionally grouped in folders
+        components/
+        # Root vue component that is mounted to the #app element by main.ts
+        App.vue
+        # The root file that mounts the App component to the DOM
+        main.ts
+        # Add syntax support in your IDE to write JSX-style typescript code (not used)
+        shims-tsx.d.ts
+        # Makes sure your IDE understands .vue files
+        shims-vue.d.ts
+    # Defines the browsers we support, e.g. used for css prefixes
+    .browserlistrc
+    # ESLint config
+    .eslintrc.js
+    # Dependency manager config (yarn is used)
+    package.json
+    # Vue cli configuration file (best to adjust webpack config here)
+    vue.config.js
+    # Versions of each dependency that was installed by yarn
+    yarn.lock
+
+# All shared scripts and assets
+shared/
```

Comment:
Shouldn't we instead have our own npm/yarn package that we can import and use instead of this shared package? We'd have 1 core library that is used by all of the frontends then. For local development we can use npm link.

Reply:
What is the difference in folder structure?

Reply:
There would be no need for a shared folder, it would become a "core" folder (or something else) that is just a package the others import.